velox/vector/arrow/Bridge.cpp

```diff
@@ -1447,6 +1451,11 @@ void exportToArrow(
           maps.getNullCount());
       exportToArrow(rows, *child, options);
       child->name = "entries";
+      // Map data should be a non-nullable struct type
+      child->flags = clearArrowNullableFlag(child->flags);
```

Comment:
Would it make sense to change clearArrowNullableFlag take flags as in/out parameter?

```
clearArrowNullableFlag(child->flags);
```

Would it also make sense to remove 'Arrow' from the function name? It seems to be implied.